resources/js/requests/components/RequestsListing.vue

```diff
@@ -141,12 +141,17 @@ export default {
           field.field = column.field;
         }
 
-        if (column.format === 'datetime') {
-        field.callback = 'formatDateUser|datetime';
+        let excludedFields = [
```

Comment:
@pmPaulis @marcoAntonioNina 

Consider not to use exceptions list. Review the line 258 in this same file because it has a similar logic:
![image](https://user-images.githubusercontent.com/8028650/182903597-80170346-f413-4436-a8c1-124637f84732.png)

Reply:
@caleeli ,
If we remove those lines the format will not work in other **core** lists:
Requests -> My Requests
Requests -> Requests In Progress
Requests -> Completed Requests
![My Requests - ProcessMaker](https://user-images.githubusercontent.com/42388723/182905930-cd4c8553-aec3-4463-ae34-d5f6833a2bea.png)
Regards,